src/com/facebook/buck/jvm/kotlin/KotlinLibraryBuilder.java

```diff
@@ -62,7 +62,10 @@ protected CompileToJarStepFactory buildCompileStepFactory() {
       return new KotlincToJarStepFactory(
           Preconditions.checkNotNull(kotlinBuckConfig).getKotlinc(),
           extraKotlincArguments,
-          EMPTY_EXTRA_CLASSPATH);
+          EMPTY_EXTRA_CLASSPATH,
+          getJavac(),
```

Comment:
Why do you provide a `javaBuckConfig` for the `KotlinAndroidLibraryBuilder` but not for the `KotlinLibraryBuilder`?